src/ExamTimer/ExamTimer.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {
+  Button,
+  Alert,
+  Icon,
+  useToggle,
+} from '@edx/paragon';
+import {
+  Visibility,
+  VisibilityOff,
+} from '@edx/paragon/icons';
+
+const ExamTimer = () => {
+  const [isShowMore, showMore, showLess] = useToggle(false);
+  const [isShowTimer, showTimer, hideTimer] = useToggle(true);
+
+  return (
+    <Alert variant="info">
+      <div className="row">
+        <div className="col col-8">
+          You are taking &#34;<Alert.Link href="#">Subsection</Alert.Link>&#34;
```

Comment:
@UvgenGen please wrap all the texts with `<FormattedMessage>` as it is done in https://github.com/raccoongang/frontend-lib-special-exams/blob/main/src/ExamInstructions/ExamInstructions.jsx